about.html

```diff
@@ -0,0 +1,106 @@
+        <li class="top-header-link"><button class="top-header-item" type="button">La Rienda</button></li>
+        <li class="top-header-link"><button class="top-header-item" type="button">Logout</button></li>
+      </ul>
+    </div>
+    <div class="main-header">
+      <div class="header-logo sm-none">
+        <a href="index.html"><img src="./images/logo.png" alt="Logo"></a>
+      </div>
+      <nav class="header-menu-button-container lg-none">
+        <button type="button" class="menu-button">&#9776;</button>
+      </nav>
+      <nav class="header-nav sm-none">
+        <ul class="header-links">
+          <li class="header-link"><a href="about.html">About</a></li>
+          <li class="header-link disabled"><a href="#">Schedule</a></li>
+          <li class="header-link disabled"><a href="#">Tickets</a></li>
+        </ul>
+      </nav>
+      <div class="menu display-none">
+        <button type="button" class="menu-close">&#10799;</button>
+        <ul class="menu-list">
+          <li><a href="."><img src="./images/logo.png" alt="Logo"></a></li>
+          <li class="menu-item"><a href="about.html">About</a></li>
+          <li class="menu-item disabled"><a href="#">Schedule</a></li>
+          <li class="menu-item disabled"><a href="#">Tickets</a></li>
+        </ul>
+      </div>
+    </div>
+  </header>
+  <main class="container">
+    <section class="headline-container">
+      <span class="headline-subtitle">"From Mendoza to the World!"</span>
+      <h1 class="headline-title">La Rienda World Tour 2023</h1>
+      <p class="headline-text">This tour was created many years ago, but with the COVID-19, it's had to be suspended, but for this years with the same enthusiasm as always. On this occasion, he will tour 3 continents, ending the tour in Japan</p>
+      <p class="headline-contact">Please contact us per Email for any further questions about La Rienda World Tour!<br><a href="#" class="headline-email">worldtour2023@larienda.com</a></p>
+    </section>
+    <section class="presentation-container">
+      <h2 class="speakers-title">History</h2>
+      <hr class="title-separator">
+      <div class="presentation-content">
+        <p class="presentation-text">La Rienda is a band born in the city of Mendoza, Argentina in 2009. It's a group of young musicians who propose a particular style having traditional dances and Argentine folk music as a common language.<br> This group has toured numerous stages throughout Argentina, winners of the Diario UNO Stage Awards for the best traditional folk group, recognized by the Honorable Chamber of Senators of the province of Mendoza for their contribution to regional culture.<br>In his repertoire sounds and melodies are combined with the authenticity of modern instruments</p>
+        <div class="presentation-image-container"><img src="./images/logo.png" alt="Logo"></div>
+      </div>
+    </section>
+    <hr class="section-separator">
+    <section class="history-container">
```

Comment:
This section of your application is overlapping on the mobile navbar menu. Have a look at this and kindly fix this error. ⬇️
![image](https://user-images.githubusercontent.com/33268927/196945266-32c2a2eb-7e51-46b3-b9b8-e5d14dc7a2f3.png)